rllib/core/learner/learner.py

```diff
@@ -1565,6 +1549,36 @@ def _make_module(self) -> MultiRLModule:
 
         return module
 
+    def rl_module_is_compatible(self, module: RLModule) -> bool:
```

Comment:
Yes, this is what we can use now where almost everything is defined by interfaces.